src/Util/PHP/DefaultPhpProcess.php

```diff
@@ -40,7 +40,7 @@ class DefaultPhpProcess extends AbstractPhpProcess
      */
     public function runJob(string $job, array $settings = []): array
     {
-        if ($this->stdin || $this->useTemporaryFile()) {
+        if ($this->stdin) {
             if (!($this->tempFile = tempnam(sys_get_temp_dir(), 'phpunit_')) ||
```

Comment:
we still need the temp-file for PHPT cases in which [STDIN is provided via test-case](https://github.com/sebastianbergmann/phpunit/commit/b9100692ac6a15c75ab2be06976c9d27de0f7bc5)